dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -436,8 +453,10 @@ internal open class RumViewScope(
                 isActive = !stopped,
                 cpuTicksCount = cpuTicks,
                 cpuTicksPerSecond = cpuTicks?.let { (it * ONE_SECOND_NS) / updatedDurationNs },
-                memoryAverage = lastMemoryVitalInfo?.meanValue,
-                memoryMax = lastMemoryVitalInfo?.maxValue
+                memoryAverage = lastMemoryInfo?.meanValue,
```

Comment:
We should make a copy of `lastMemoryInfo` first here in order to make sure this value is not changed while computing the `refreshRateAverage`, `refreshRateMin`.